projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad.php

```diff
@@ -155,6 +160,7 @@ public function get_data( $request ) {
 			'is_enabled'         => wpcom_get_launchpad_task_list_is_enabled( $checklist_slug ),
 			'is_dismissed'       => wpcom_launchpad_is_task_list_dismissed( $checklist_slug ),
 			'is_dismissible'     => wpcom_launchpad_is_task_list_dismissible( $checklist_slug ),
+			'is_dismissed_until' => wpcom_launchpad_task_list_dismissed_until( $checklist_slug ),
```

Comment:
I think we can simplify this by adding the `is_dismissed_until` functionality to the `wpcom_launchpad_is_task_list_dismissed` function. Meaning that we could remove this new prop and just use the `is_dismissed`

Reply:
As we chat about, I think is very weird to update a resource and not see it on the resource response, it maybe make the api usage and test bit harder.

Reply:
While I agree that it's weird to not see the result in the response I can't think of any usage for this prop other than testing, which can also be done by looking at the actual `launchpad_checklist_tasks_statuses` option value.
It's probably a good idea to remove this until/if we find a worthwhile use case for having it.

Reply:
My reasoning is that since we are not using it, we could save some processing if we remove that call. Going even further, perhaps we could even make an early return in the case the Launchpad is dismissed, and prevent invoking all these functions.

Reply:
But this is a nit issue. Feel free to ignore it!

Reply:
### For context
At first, I misunderstood how the check for the dismissal was being done. I thought we used `is_dismissed_until` on the front end to check it. But in the sync call, we discussed it, and Gabriel explained that the check for the dismissal is indeed in the `wpcom_launchpad_is_task_list_dismissed` function. The conversation then changed to whether we should have the unused `is_dismissed_until` returned in the endpoint.

Reply:
I am a fan of not adding something if you are not going to use it, especially when it's easy to add down the road. I don't think this will hurt performance, but it's more code to maintain.

Reply:
@andres-blanco @valterlorran  removed [26f541c](https://github.com/Automattic/jetpack/pull/34889/commits/26f541cbd7d638a87de3128fbde95087380236e9)